src/software/guides/addin-icon-validator.html

```diff
@@ -0,0 +1,325 @@
+        }
+
+        .logo-wrapper-neutral svg polyline{
+            fill: #66788C !important;
+            stroke: none !important;
+        }
+
+        .logo-wrapper-neutral svg rect{
+            fill: #66788C !important;
+            stroke: none !important;
+        }
+
+        .logo-wrapper-neutral svg text{
+            fill: #66788C !important;
+            stroke: none !important;
+        }
+
+        /* SELECTED */
+        .logo-wrapper-selected svg circle{
+            fill: #FFFFFF !important;
+            stroke: none !important;
+        } 
+
+        .logo-wrapper-selected svg ellipse{
+            fill: #FFFFFF !important;
+            stroke: none !important;
+        } 
+
+        .logo-wrapper-selected svg line{
+            fill: #FFFFFF !important;
+            stroke: none !important;
+        }  
+        
+        .logo-wrapper-selected svg path{
+            fill: #FFFFFF !important;
+            stroke: none !important;
+        }  
+        
+        .logo-wrapper-selected svg polygon{
+            fill: #FFFFFF !important;
+            stroke: none !important;
+        }  
+    
+        .logo-wrapper-selected svg polyline{
+            fill: #FFFFFF !important;
+            stroke: none !important;
+        }  
+        
+        .logo-wrapper-selected svg rect{
+            fill: #FFFFFF !important;
+            stroke: none !important;
+        }  
+        
+        .logo-wrapper-selected svg text{
+            fill: #FFFFFF !important;
+            stroke: none !important;
+        }  
+        </style>
+        
+    </head>
+
+    <body>
+        <div class="banner">
+            <h1>MyGeotab SVG Icon Tester</h1>
+            <a href="https://www.geotab.com/blog/mygeotab-add-in-icons-specs/" id="specificationsLink" target="_blank">Link to Add-In Icon Specifications</a>
+        </div>    
+        
+        <br>
+
+        <form name="uploadForm">
+          
+          <div id="parent" class="container">
+            <div class="item">
+
+                <div class="inline-buttons">
+                    <label for="svgUpload">
+                        <span class="intgr-button intgr-button--primary" aria-hidden="true">Choose File</span>
+                        <input id="svgUpload" onChange="handleFiles()" type="file" name="svgUpload" style="display:none"><br>
+                    </label>
+                    
+                </div>
+
+                <div class="intgr-button-switcher intgr-button-switcher--horizontal intgr-button-switcher--short-list">
+                    <input type="radio" name="displayed" class="" id="neutral" checked onChange="handleFiles()">
+                    <label class="" for="neutral">Neutral</label>
+                    <input type="radio" name="displayed" id="selected" class="" onChange="handleFiles()">
+                    <label class="" for="selected">Selected</label>
+                </div>
+
+            </div>
+
+        </form>
+        
```

Comment:
Could you double-check if there's a missing opening \<form\> tag?

Reply:
open form tag is on line 162 

<form name="uploadForm">

Reply:
But you have another close form tag in line 323

Reply:
oh yes my mistake, just removed that it was not needed